react/w-books/src/app/components/Card/index.tsx

```diff
@@ -0,0 +1,62 @@
+import clsx from 'clsx';
+import React from 'react'
+import CardProp, { iCardProp } from '../CardProp';
+
+import styles from './styles.module.scss';
+
+export interface iCard {
+  imageUrl: string;
+  id: number;
+  title: string;
+  subtitle: string;
+  properties: iCardProp[];
+}
+interface CardProps extends iCard {
+  isSimple?: boolean;
+}
+
+function Card({
+  isSimple,
+  imageUrl,
+  id,
+  title,
+  subtitle,
+  properties
+}: CardProps) {
+  return (
+    <a className={
```

Comment:
use anchor only to external links, instead of use Link component of the react router